tests/wdio.conf.js

```diff
@@ -240,6 +241,11 @@ const baseConfig = {
    * Function to be executed after a test (in Mocha/Jasmine).
    */
   afterTest: async function (test, context, { passed }) {
+    const feedBackDocs = await browserUtils.feedBackDocs(`${test.parent} ${test.title}`);
+    if(feedBackDocs){
+      passed = false;
+      context.test.callback(new Error('Feedback docs were generated during the test.'));
```

Comment:
The test might not be failing at this point. I wrote it this way under the assumption that a test could pass but we are generating errors that are logged in feedback docs.  This is the best hook I believe to run after every test without adding a bunch of after hooks.

Reply:
Yes, it's definitely the right way to do it. But if the test *is* failing then I don't think we want to call the callback. It'll either error out because the test is failed twice, or it'll blow away the original failure.

I'm suggesting wrapping this in a `if (passed) {` check so it'll fail the test if it's not already failing.

Reply:
I think I understand what you mean now. Took me a couple reads to comprehend. I'll push a change in a sec.